codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -1266,24 +1243,6 @@ impl GqlScalar {
             name => GqlScalar::Custom(name.to_string()),
         }
     }
-    fn to_postgres_type(&self, schema: &Schema) -> anyhow::Result<String> {
```

Comment:
This function is completely unused, so I removed all references to it and everything is still working correctly.